### DISCUSSION TITLE
How can i install package fron Controller ?

### DISCUSSION THREAD

**Discussion by Viruscom** [nWidart/laravel-modules#1351] [Q&A]
Hello,
How can i install package fron Controller ?

When i try with 
`Artisan::call('module:install vendorName/' . strtolower($moduleName) . '-module');`
 i get Nwidart\Modules\Exceptions\ModuleNotFoundException  Module [BlogModule] does not exist!

When i try from CLI - everything is ok.

Thank you in advance.

**Comment by AlexisVS** [comment]
Maybe you can try with Illuminate\Support\Facades\Process ?
